gyp/output/lua/jni/Application.mk

```diff
@@ -0,0 +1,38 @@
+# select target arch
+#APP_ABI := armeabi armeabi-v7a x86 mips
+#APP_ABI := armeabi
+#APP_ABI := armeabi-v7a
+APP_ABI := armeabi-v7a
+#APP_ABI := x86
+
+# CLANG versions available
+#NDK_TOOLCHAIN_VERSION := clang3.3
+NDK_TOOLCHAIN_VERSION := clang3.5
+# GCC versions available
+#NDK_TOOLCHAIN_VERSION := 4.6
+#NDK_TOOLCHAIN_VERSION := 4.8
+
+
+# target android platform
+# min version for pthread_rwlock_t
+APP_PLATFORM := android-9
+#APP_PLATFORM := android-18
```

Comment:
android 18 is what the andrive is build for, lua's min framework for correct compile is 9. so no need to raise it here. the andrive project can set it up to higher level to override this setting.